packages/autop/src/test/index.test.js

```diff
@@ -502,3 +503,12 @@ test( 'that autop correctly adds a start and end tag when followed by a div', ()
 
 	expect( autop( content ).trim() ).toBe( expected );
 } );
+
+describe( 'removep', () => {
+	test( 'preserves paragraphs with attributes', () => {
```

Comment:
It's a bit scary that this is the only `removep` test for such a huge pile of regular expressions.

Reply:
> It's a bit scary that this is the only `removep` test for such a huge pile of regular expressions.

It is, yes. For historical context, the module originated as a direct port of the PHP implementation of `autop`, including its tests. There is no `removep` in PHP, so there are no tests. There was a previous `removep` implementation in JavaScript from which this implementation was adapted, but it had no tests to port over.

I can plan to create a task issue to follow-up on adding more tests here.

Reply:
> I can plan to create a task issue to follow-up on adding more tests here.

See #15152